include_core/omrport.h

```diff
@@ -2115,6 +2116,8 @@ typedef struct OMRPortLibrary {
 	intptr_t (*sysinfo_get_CPU_load)(struct OMRPortLibrary *portLibrary, double *load) ;
 	/** see @ref omrsysinfo.c::omrsysinfo_get_processes "omrsysinfo_get_processes" */
 	uintptr_t (*sysinfo_get_processes)(struct OMRPortLibrary *portLibrary, OMRProcessInfoCallback callback, void *userData);
+	/** see @ref omrsysinfo.c::omrsysinfo_get_process_id_name "omrsysinfo_get_process_id_name"*/
+	const char *(*sysinfo_get_process_id_name)(struct OMRPortLibrary *portLibrary, uintptr_t pid) ;
```

Comment:
The return type should be `char *` because the caller is expected to free that memory.
Please remove the space before `;`.
As before, I don't think `_id_` makes sense here processes run programs, but IDs don't have names.